Source/Infrastructure/deploy/azure-secret.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: azure-storage-secret
+type: Opaque
+data:
+  azurestorageaccountname: Y2JzMmRldg==
+  azurestorageaccountkey: UmtpcUhsa3dsSnJhVUF4MW16OVBHdUoyUHQzejVEekU5eUk4ZXFTUVUzQVVzeTF6ZHBIbk0xN25NWGZ4OE50RGhwcVJSMWU2Mk91K0VVbFdMWjZLbmc9PQ==
```

Comment:
I'm a little concerned about the security here, could we store it in Azure KeyVault or similar and retrieve the key from there?

Reply:
Yup.. This is very temporary - and absolutely madness of me to do. :).  I will delete the entire storage account in a couple of days - its sitting in a very different subscription. 
We should at the very minimum be using Kubernetes secrets for this and keep them outside the repo.